source/cloud-security/amazon/services/supported-services/custom-buckets.rst

```diff
@@ -0,0 +1,133 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Learn how to configure Amazon Custom Logs Buckets fetching.
+
+.. _amazon_custom_logs:
+
+Custom Logs Buckets
+===================
+
+.. versionadded:: 4.7.0
+
+`Amazon Simple Queue Service (Amazon SQS)  <https://aws.amazon.com/sqs/>`_ is a fully managed message queuing service that offers secure, durable, and available hosted queues to decouple and scale software systems and components.
+It allows sending, storing, and receiving messages between software components at any volume, without losing messages or requiring other services to be available. These features make it an optimal component to associate with Amazon S3 Buckets to consume any type of log.
+
+Combining Amazon SQS with Amazon S3 buckets, allows Wazuh to fetch any JSON, CSV or plain text logs, originated in AWS or not, and process the events inside them.
+
+.. note::
+  It is a precondition of the module for the CSV logs to have the corresponding column headers to be properly processed.
+
+To set up the Wazuh integration for Custom Logs Buckets, you need to do the following:
+
+    #. Create an AWS SQS Queue.
+    #. Configure an S3 bucket to send notifications to the queue for every object creation event.
+
+AWS configuration
+-----------------
+
+Amazon Simple Queue Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set up a **Standard** type SQS Queue with the default configurations and apply the following **Access Policy**:
+
+.. code-block:: json
+
+  {
+  "Version": "2012-10-17",
+  "Id": "example-ID",
+  "Statement": [
+    {
+      "Sid": "example-access-policy",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "s3.amazonaws.com"
+      },
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:<region>:<account-id>:<s3-bucket>",
+      "Condition": {
+        "StringEquals": {
+          "aws:SourceAccount": "<account-id>"
+        },
+        "ArnLike": {
+          "aws:SourceArn": "arn:aws:s3:*:*:<s3-bucket>"
+        }
+      }
+    }
+  ]
+  }
+
+  
+.. note::
+     This is an example Access Policy. It can be modified to accept S3 notifications from different ``<account-id>`` values or to apply different conditions. More information `here <https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-overview-of-managing-access.html>`_. 
+
+Amazon S3 and Event Notifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The steps to have an S3 bucket reporting creation events are:
+
+#. Configure an S3 bucket as defined in the :ref:`Configuring an S3 Bucket <s3_bucket>` section with the name provided in the previous section.
```

Comment:
What previous section? What name provided?

Reply:
It probably refers to the `<s3-bucket>` in [this line](https://github.com/wazuh/wazuh-documentation/blob/491e0b6b7973567a28f9308e9dba10d07357892e/source/cloud-security/amazon/services/supported-services/custom-buckets.rst?plain=1#L47C54-L47C66) and [this one](https://github.com/wazuh/wazuh-documentation/blob/491e0b6b7973567a28f9308e9dba10d07357892e/source/cloud-security/amazon/services/supported-services/custom-buckets.rst?plain=1#L53C43-L53C55).